pkg/flowkit/services/project.go

```diff
@@ -92,6 +95,131 @@ func (p *Project) Init(
+		},
+		"FlowToken": {
+			Name:     "FlowToken",
+			Address:  flow.HexToAddress("0x1654653399040a61"),
+			InfoLink: "https://developers.flow.com/flow/core-contracts/flow-token",
+		},
+		"FlowFees": {
+			Name:     "FlowFees",
+			Address:  flow.HexToAddress("0xf919ee77447b7497"),
+			InfoLink: "https://developers.flow.com/flow/core-contracts/flow-fees",
+		},
+		"FlowServiceAccount": {
+			Name:     "FlowServiceAccount",
+			Address:  flow.HexToAddress("0xe467b9dd11fa00df"),
+			InfoLink: "https://developers.flow.com/flow/core-contracts/service-account",
+		},
+		"FlowStorageFees": {
+			Name:     "FlowStorageFees",
+			Address:  flow.HexToAddress("0xe467b9dd11fa00df"),
+			InfoLink: "https://developers.flow.com/flow/core-contracts/service-account",
+		},
+		"FlowIDTableStaking": {
+			Name:     "FlowIDTableStaking",
+			Address:  flow.HexToAddress("0x8624b52f9ddcd04a"),
+			InfoLink: "https://developers.flow.com/flow/core-contracts/staking-contract-reference",
+		},
+		"FlowEpoch": {
+			Name:     "FlowEpoch",
+			Address:  flow.HexToAddress("0x8624b52f9ddcd04a"),
+			InfoLink: "https://developers.flow.com/flow/core-contracts/epoch-contract-reference",
+		},
+		"FlowClusterQC": {
+			Name:     "FlowClusterQC",
+			Address:  flow.HexToAddress("0x8624b52f9ddcd04a"),
+			InfoLink: "https://developers.flow.com/flow/core-contracts/epoch-contract-reference",
+		},
+		"FlowDKG": {
+			Name:     "FlowDKG",
+			Address:  flow.HexToAddress("0x8624b52f9ddcd04a"),
+			InfoLink: "https://developers.flow.com/flow/core-contracts/epoch-contract-reference",
+		},
+		"NonFungibleToken": {
+			Name:     "NonFungibleToken",
+			Address:  flow.HexToAddress("0x1d7e57aa55817448"),
+			InfoLink: "https://developers.flow.com/flow/core-contracts/non-fungible-token",
+		},
+		"MetadataViews": {
+			Name:     "MetadataViews",
+			Address:  flow.HexToAddress("0x1d7e57aa55817448"),
+			InfoLink: "https://developers.flow.com/flow/core-contracts/nft-metadata",
+		},
+	}
+
+	contracts, err := p.state.DeploymentContractsByNetwork("mainnet")
+	if err != nil {
+		return err
+	}
+
+	for _, contract := range contracts {
+		standardContract, ok := mainnetContracts[contract.Name]
+
+		if !ok {
+			continue
+		}
+
+		p.logger.Info(fmt.Sprintf("It seems like you are trying to deploy %s to Mainnet \n", contract.Name))
+		p.logger.Info(fmt.Sprintf("It is a standard contract already deployed at address 0x%s \n", standardContract.Address.String()))
+		p.logger.Info(fmt.Sprintf("You can read more about it here: %s \n", standardContract.InfoLink))
+
+		useMainnetVersionPrompt := promptui.Select{
+			Label: "Do you wish to use Mainnet version instead? (y/n)",
+			Items: []string{"Yes", "No"},
+		}
+		_, useMainnetVersion, err := useMainnetVersionPrompt.Run()
+		if err == promptui.ErrInterrupt {
+			os.Exit(-1)
+		}
+
+		if useMainnetVersion == "Yes" {
+			err := p.ReplaceStandardContractReferenceToAlias(standardContract)
+			if err != nil {
+				return err
+			}
+		}
```

Comment:
could we extract this to just return to a function in prompts.go and just return bool.